cinder-{{cookiecutter.driver_name_lc}}/tests/bundles/focal-{{cookiecutter.release}}.yaml

```diff
@@ -0,0 +1,76 @@
+series: focal
+variables:
+  openstack-origin: &openstack-origin distro
+comment:
+- 'machines section to decide order of deployment. database sooner = faster'
+machines:
+  '0':
+    constraints: mem=3072M
+  '1':
+    constraints: mem=3072M
+  '2':
+    constraints: mem=3072M
+  '3':
+  '4':
+  '5':
+    constraints: mem=4G root-disk=16G
+local_overlay_enabled: false
+relations:
+  - - keystone:shared-db
+    - keystone-mysql-router:shared-db
+  - - keystone-mysql-router:db-router
+    - mysql-innodb-cluster:db-router
+  - - cinder:shared-db
+    - cinder-mysql-router:shared-db
+  - - cinder-mysql-router:db-router
+    - mysql-innodb-cluster:db-router
+  - - cinder:identity-service
+    - keystone:identity-service
+  - - cinder:amqp
+    - rabbitmq-server:amqp
+  - - cinder:storage-backend
+    - cinder-{{ cookiecutter.driver_name_lc }}:storage-backend
+applications:
+  mysql-innodb-cluster:
+    charm: cs:~openstack-charmers-next/mysql-innodb-cluster
```

Comment:
this needs to be updated to charmhub references, but that can be done in a subsequent PR.